R/volcanoPlot.R

```diff
@@ -46,36 +46,30 @@
 #'    not in topTable output by default so the user has to bind this column
 #'    to the dataframe in advance.  This column will be used to label
 #'    significantly changed points.
-#' @param pthresholdLine Color for a horizontal line at the p-threshold (Default
-#'   = NULL (disabled))
-#' @param symbolSize Size of symbols for Up, no change, and Down. default = c(4,
-#'        3.99, 4); Note: All three cannot be the same size. Decimal values are acceptable to help offset that
+#' @param symbolSize Size of symbols for Up, no change, and Down. default = c(10, 4, 10);
```

Comment:
Did you rename the pthresholdLine to reference line ? If yes, can you please revert it back.

Reply:
From old function of volcano plot the pthresholdLine is horizontal line which is lying on Yaxis. and Its value is 
-log10(pthreshold).

Can I proceed as same for CX or it will be vertical line because for volcano plots we using lines as reference in vertical only.

Reply:
completed